server/app/facets/mediawiki-page.js

```
@@ -88,43 +91,45 @@ function handleResponse(request, reply, data, allowCache = true, code = 200) {
 	let result = {},
 		pageData = {},
 		viewName = 'wiki-page',
-		response,
-		ns;
+		isMainPage = false,
+		isContentNamespace,
+		ns,
+		response;
 
 	if (data.page && data.page.data) {
 		pageData = data.page.data;
 		ns = pageData.ns;
-		result.mediaWikiNamespace = ns;
+		isMainPage = pageData.isMainPage;
 	}
+
+	result.mediaWikiNamespace = ns;
```

If yes then which one? `0`? Then we might get caught trying to display a non-content namespace as a such. If we have no data, we can throw an error on line 126.